Cirumflex_Combined.ino

```diff
@@ -1,15 +1,25 @@
+// Uncomment to turn on, Comment to turn off debug() output:
+//#define DEBUG
+#include "debug.h"
```

Comment:
I provided a mechanism to print stuff during debugging, easily turned off. We'll see an example below.